### DISCUSSION TITLE
Is Node 18 really required?

### DISCUSSION THREAD

**Discussion by axelboc** [ljosberinn/eslint-config-galex#752] [General]
pnpm is telling me that my Node version (16) is incompatible with the one specified in galex' `package.json`. I don't mind upgrading in some evergreen projects, but in others it will take more time and convincing...

I assume this is a development requirement, not a requirement of the galex library per se? Would it be possible to update the `package.json` published to NPM to remove the `engines` field, maybe?

**Comment by ljosberinn** [comment]
hey, good point. I initially wanted to upgrade .. I think read-pkg-up to latest but then realized while I can install and import it, it'll still have to run in a CJS context due to ESLint itself, so I downgraded again, but kept the Node version since it's LTS: https://nodejs.org/en

I'll check whether I can drop the field and republish, yea.

**Reply by ljosberinn** [reply]
ah semantic-release wants it too

**Reply by ljosberinn** [reply]
semantic-release wants this since v20 already, meaning the Node 18 requirement isnt even new, but from January 👀

**Reply by ljosberinn** [reply]
I'll downgrade to the latest non-ESM version regardless as it's dev/CI related only however

**Reply by axelboc** [reply]
Awesome, thanks for the reactivity as always :blush:
